tests/aead_tests.rs

```diff
@@ -128,7 +128,7 @@ fn test_aead(aead_alg: &'static aead::Algorithm, file_path: &str) {
             257, // 16 AES blocks or 8 ChaCha20 blocks, plus 1.
         ];
 
-        let mut more_comprehensive_in_prefix_lengths = [0; 4096];
+        let mut more_comprehensive_in_prefix_lengths = [0xDE; 4096];
```

Comment:
Instead of hard-coding the `0xDE` constant here, please give it a name with `const UNINITIALIZED: u8 = 0xDE;` and then use `UNINITIALIZED` everywhere.

Reply:
Oh, wait. I see that you chose a different value for each test. I can see the value in that. However, it's also more confusing for the reader because they need to understand the significance of that constant. Maybe it's better to use the same non-zero named and well-documented constant in all tests?